### PR TITLE
List Viacoin (VIA)

### PR DESCRIPTION
**Pull request by romanornr**
Official project URL: https://viacoin.org
Official block explorer URL: https://explorer.viacoin.org

third party block explorer URL: https://chainz.cryptoid.info/via/

### CODE REVIEW

File: src/test/java/bisq/asset/coins/ViacoinTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    public ViacoinTest() {
+        super(new Viacoin());
+    }
+
+    @Test
+    public void testValidAddresses() {
+        assertValidAddress("VdMPvn7vUTSzbYjiMDs1jku9wAh1Ri2Y1A");
+        assertValidAddress("EPt5g6LookySkqUNTwky24wTvzNrmKDP7r");
+        assertValidAddress("EVt5ypcAfiCccFrdHxPbFwnEo5WDevawM7");
+        //assertValidAddress("via1q95qlu98cpj23xy6w9tdnfn65n5vkpkey99g6wl"); bech32 for in the future
```
Comment (blabno) [inline src/test/java/bisq/asset/coins/ViacoinTest.java]: Remove commented out code.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: src/test/java/bisq/asset/coins/ViacoinTest.java (hunk 1 above)
Comment (romanornr) [reply src/test/java/bisq/asset/coins/ViacoinTest.java]: I removed the commented out coded.

**Comment by romanornr** [comment]
I removed the comments.

**Comment by romanornr** [comment]
Any update about this?

**Review by cbeams — APPROVED** [review]
ACK. Note that merges are on hold until the resolution of https://github.com/bisq-network/proposals/issues/35. Please track that issue to stay up to date.

**Comment by cbeams** [comment]
This repository has been moved into the 'assets' directory of the https://github.com/bisq-network/bisq repository. Please resubmit this pull request against that repository. See https://github.com/bisq-network/bisq-docs/pull/84 for updated instructions. When submitting your pull request, please mention in the description (a) that this is a resubmission, (b) paste a link back to this original pull request, and (c) whether the original had already bee ACKed. Thank you.

**Pull request closed without merging**